### PR TITLE
Rahul

### PR DESCRIPTION
**Pull request by Rahul-Chimple**

### CODE REVIEW

File: src/pages/StudentProgress.tsx [hunk 1]
```diff
@@ -236,22 +241,37 @@ const StudentProgress: React.FC = () => {
...
+        tabNames={["English", "Maths", "Digital Skills"]}
+        value={tabIndex}
+        onChange={handleChange}
+      />
 
-          <div>{displayProgressUI(currentHeader)}</div>
-        </div>
-      ) : null}
-      <Loading isLoading={isLoading} />
+      {tabIndex === "English" && (
```
Comment (Sutara-Skanda) [inline src/pages/StudentProgress.tsx:261]: Make tabIndex elements also dynamic

File: src/pages/StudentProgress.tsx [hunk 2]
```diff
@@ -236,22 +241,37 @@ const StudentProgress: React.FC = () => {
...
-      {!isLoading ? (
-        <div id="student-progress">
-          <StudentProgressHeader
-            currentHeader={currentHeader}
-            headerIconList={studentProgressHeaderIconList}
-            onHeaderIconClick={onHeaderIconClick}
-          ></StudentProgressHeader>
+<div>
+      <StudentProgressHeaderTabBar
+        tabNames={["English", "Maths", "Digital Skills"]}
```
Comment (Sutara-Skanda) [inline src/pages/StudentProgress.tsx:256]: Get this course List from User's Object

File: src/components/studentProgress/StudentProgressHeaderTabBar.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    onChange(newValue);
+  };
+  const history: any = useHistory();
+
+  return (
+    <AppBar position="static" sx={{ flexDirection: "inherit", justifyContent: "space-between", padding: "1vh 1vw", backgroundColor: "#FF7925 !important", boxShadow: "0px 0px 0px 0px !important" }}>
+      <BackButton
+            // iconSize={"8vh"}
+            onClicked={() => {
+              history.replace(PAGES.PARENT);
```
Comment (Sutara-Skanda) [inline src/components/studentProgress/StudentProgressHeaderTabBar.tsx:25]: Convert this function as callback function

File: src/components/parent/RectangularIconButton.tsx [hunk 4]
```diff
@@ -17,8 +19,16 @@ const RectangularIconButton: React.FC<{
   isButtonEnable,
   onHeaderIconClick,
 }) => {
+
   // const isButtonEnable;
+  const [tabIndex, setTabIndex] = useState(name);
+
+  const handleChange = (event: React.SyntheticEvent, newValue: string) => {
+    // setValue(newValue);
```
Comment (Sutara-Skanda) [inline src/components/parent/RectangularIconButton.tsx:27]: remove tabIndex and handlechange incase you are not using anywhere

File: src/components/parent/RectangularIconButton.tsx [hunk 5]
```diff
@@ -29,17 +39,17 @@ const RectangularIconButton: React.FC<{
         opacity: isButtonEnable ? "1" : "0.5",
       }}
     >
-      <img
+       {/* { <img
```
Comment (Sutara-Skanda) [inline src/components/parent/RectangularIconButton.tsx:42]: Why you commented Image element here????

**Review by Sutara-Skanda — CHANGES_REQUESTED** [review]

**Comment by bhanugummana** [comment]
@Rahul-Chimple  Explain About this PR in title and description

**Pull request closed without merging**